docs/pages/plugins-releases-55.md

```diff
@@ -0,0 +1,199 @@
+
+function handleURL(context) {
+  let query = context.actionContext.query
+  if (currentDocument) {
+    sketch.UI.message(query.foo)
+  } else {
+    let document = new Document()
+    sketch.UI.message(query.foo)
+  }
+}
+```
+
+The full plugin example can be found [here](https://github.com/KevinGutowski/HandleURL_Example).
+
+### Add `isSelected` method on a `CurvePoint`
+
+[_GitHub pull request_](https://github.com/BohemianCoding/SketchAPI/pull/445)
+
+In case the user is currently editing a path, you can check if a curve point is selected using the `curvePoint.isSelected()` method.
+
+#### Usage
+
+If the user is selecting a point of a shape you can check it with:
+
+```js
+shape.points[0].isSelected()
+```
+
+Here is an example for how you might use it in practice. Create and select a rectangle in your Sketch document.
+
+```js
+let sketch = require('sketch')
+let document = sketch.getSelectedDocument()
+let selection = document.selectedLayers.layers[0]
+
+let isAnyPointSelected = false
+let pointSelectionArray = []
+
+selection.points.forEach(point => {
+  if (point.isSelected()) {
+    isAnyPointSelected = true
+    pointSelectionArray.push(true)
+  } else {
+    pointSelectionArray.push(false)
+  }
+})
+
+console.log(isAnyPointSelected)
+// true
+
+console.log(pointSelectionArray)
+// [true, true, false, false]
+```
+
+### `getSelectedDocument()` previously threw an error without an open document, now returns `undefined` instead
+
+[_GitHub pull request_](https://github.com/BohemianCoding/SketchAPI/pull/459)
+
+Babel would tranform `[nativeDocument] = NSApplication.sharedApplication().orderedDocuments()` assuming that it's a proper array but it's not, it's an `NSArray` so it would throw an error.
+
+This is a great addition with the URL example from earlier. We can confidently check if there is a current document open or not.
+
+#### Usage
+
+```js
+let sketch = require('sketch')
+let document = sketch.getSelectedDocument()
+
+if (document) {
+  // has open document
+}
+```
+
+#### Improve consistency by deprecating `Fill.fill` in favor of `Fill.fillType`
+
+[_GitHub pull request_](https://github.com/BohemianCoding/SketchAPI/pull/463)
+
+This was done to match `Border.fileType` and other types.
+
+#### Usage
+
+Setting a fill is more consistent to setting a border.
+
+##### Old
+
+```js
+const style = new Style({
+  fills: [{ color: '#1234', fill: Style.FillType.Color }],
+})
+```
+
+##### New
+
+```js
+const style = new Style({
+  fills: [{ color: '#123456', fillType: Style.FillType.Color }],
+})
+```
+
+### Some better logging of the prototype of wrapped objects
+
+[_GitHub pull request_](https://github.com/BohemianCoding/SketchAPI/pull/451)
+
+There was a bug in the `util.inspect` algorithm (which console uses) that makes it think the prototype of a wrapped object is a wrapped object and uses the code path for wrapped object. This has been fixed for better logging output.
+
+### Upcoming Additions
```

Comment:
not sure we should put this here, it will be for 56 ;)

Reply:
Yeah, ok. Let's take it out